### PR TITLE
i3-readme

### PR DESCRIPTION
**Pull request by ryandbarnett**
Added a basic readme template. 

The reflection at the bottom should be answered individually after the project is over. We can all fork this project to our own repositories and then fill it out.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,85 @@
-# Futbol
+# Futbol :soccer:
 
-Starter repository for the [Turing School](https://turing.io/) Futbol project..
+## Table of Contents
```
Comment (dhalverson) [inline README.md:3]: This looks great!

**Review by dhalverson — APPROVED** [review]
This looks good as a Readme starter

File: README.md (hunk 1 above)
Comment (ryandbarnett) [reply README.md:3]: Thanks! The table of contents might be a bit much for our project but I threw it in anyway.

Comment (hashmaster3k) [reply README.md:3]: Wow this is a phenomenal start!

Comment (ryandbarnett) [reply README.md:3]: Thanks!
I googled great personal project readme template and a [Turing template](https://gist.github.com/martensonbj/6bf2ec2ed55f5be723415ea73c4557c4) came up first lol.

**Review by hashmaster3k — CHANGES_REQUESTED** [review]
Wow! Lets just get those blanks filled in and we're set.

**Review by hashmaster3k — APPROVED** [review]

**Pull request merged**